weave/tests/trace/test_client_trace.py

```diff
@@ -2297,6 +2297,58 @@ def test_obj(val):
         assert inner_res.count == count
 
 
+def test_in_operation(client):
+    @weave.op()
+    def test_op(label, val):
+        return val
+
+    test_op(1, [1, 2, 3])
+    test_op(2, [1, 2, 3])
+    test_op(3, [5, 6, 7])
+    test_op(4, [8, 2, 3])
+
+    call_ids = [call.id for call in test_op.calls()]
+    assert len(call_ids) == 4
+
+    strs = [str(call_id) for call_id in call_ids]
```

Comment:
It looks like call.id can be None (though I'm not sure when that can occur) but if not None it is already a `str`, so not sure what this is here for.

Reply:
I was getting a weird boxed_str pydantic warning, but now that appears to be resolved so i will revert, thx.